src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -448,17 +472,28 @@ def get_webhook_data(
 def send_resource_change_webhook(
     installation_id: int, event: str, data: dict[str, Any], *args: Any, **kwargs: Any
 ) -> None:
-    installation = app_service.installation_by_id(id=installation_id)
-    if not installation:
-        logger.info(
-            "send_resource_change_webhook.missing_installation",
-            extra={"installation_id": installation_id, "event": event},
-        )
-        return
+    with SentryAppInteractionEvent(
+        operation_type=SentryAppInteractionType.SEND_WEBHOOK, event_type=event
+    ).capture() as lifecycle:
+        installation = app_service.installation_by_id(id=installation_id)
+        if not installation:
+            error = SentryAppSentryError(
+                message=f"{SentryAppWebhookFailureReason.MISSING_INSTALLATION}",
+                webhook_context={"installation_id": installation_id, "event": event},
+            )
+            sentry_sdk.capture_exception(error)
+            raise error
 
-    send_webhooks(installation, event, data=data)
+        try:
+            send_webhooks(installation, event, data=data)
+        except SentryAppSentryError as e:
+            sentry_sdk.capture_exception(e)
+            raise
```

Comment:
```suggestion
```
you shouldn't need to capture and reraise, the exception not being caught in the first place will generate a Sentry error

Reply:
retry doesnt send error to sentry for ignored errors :C

Reply:
TIL